### PR TITLE
Codebase cleanup

### LINKED ISSUE #5: As a developer, I want the code base to be clear, concise, and organized.
- [x] Exonoesis sweep of all files
- [x] Divark sweep of all files

### PR DESCRIPTION
**Pull request by Exonoesis**
This completes a much-needed general cleanup of the codebase and closes #5

### CODE REVIEW

File: Assets/Scripts/Controllers/SceneNavigator.cs [hunk 1]
```diff
@@ -51,7 +46,7 @@ public void changeSceneIfSatisfied(Asset clickedAsset)
 
     private void showErrorScene(Asset clickedAsset, Conditional failedCondition)
     {
-        Scene errorToShow = _errorSceneTable[(clickedAsset, failedCondition)];
+        Cutscene errorToShow = _errorSceneTable[(clickedAsset, failedCondition)];
 
         _currentScene.hide();
         setCurrentScene(errorToShow);
```
Comment (divark) [inline Assets/Scripts/Controllers/SceneNavigator.cs:52]: Given that `showErrorScene` and `showNextScene` are very similar but with one line differences, the two methods can be merged into a method called `showScene`, as follows:
```csharp
private void showScene(Scene newScene)
{
    _currentScene.hide();
    setCurrentScene(newScene);
    newScene.show();
}
```

turning lines 39-40 and 44-45 to something like
```csharp
if (!condition.isMet())
{
    Cutscene errorToShow = _errorSceneTable[(clickedAsset, failedCondition)];
    showScene(errorToShow);
    return;
}

...
showNextScene(clickedAsset.getState().getNextScene());
```

Thoughts?

**Review by divark — CHANGES_REQUESTED** [review]
Nice job making the codebase cleaner! I just have one concern regarding a possible method merging.

File: Assets/Scripts/Controllers/SceneNavigator.cs (hunk 1 above)
Comment (Exonoesis) [reply Assets/Scripts/Controllers/SceneNavigator.cs:52]: Oh, I like that idea.

Comment (Exonoesis) [reply Assets/Scripts/Controllers/SceneNavigator.cs]: I've included this change in the most recent commit.

**Pull request merged**